editor/src/components/custom-code/code-file.test-utils.ts

```diff
@@ -80,3 +86,24 @@ export function getTextFileByPath(projectContents: ProjectContentTreeRoot, path:
     throw new Error(`Unable to find a text file at path ${path}.`)
   }
 }
+
+export function getCodeForFile(
```

Comment:
I'd suggest renaming this to reflect the fact that this isn't just getting the code, but is actually printing the parsed model

Reply:
Oh sorry, I hadn't realised this was just moved from a different file. I think it's still worth renaming though